### DISCUSSION TITLE
Can you please help me urgent . For 1.0 please

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#128] [Q&A]
I want to make an app. When you are on home page it will start the 5 min timer after 5 min timer will redirect your to the page 2 forever which means every time I open the app it will always open page 2 instead of homepage . And the timer would run in background too. 

Please give me the code . Please paste the code here please . Its better if you past it here because from which I learn a lot .

**Comment by XiaomaiTX** [comment]
I'll provide an idea, you can try to use the file system and judgment in the home page.

When you enter the applet for the first time, determine whether there is a file named "config.json" in the /data directory, if not, it is the first time you enter the applet, then you can create a new file "config.json" and create a timer.

When you enter the applet for the second time, determine whether the file named "config.json" exists in the /data directory, and if it does, jump to page 2 directly.

Here is the reference document of the file system
https://docs.zepp.com/docs/reference/device-app-api/newAPI/fs/openSync/

A better implementation is data persistence, which can be found in the following document
https://docs.zepp.com/docs/guides/best-practice/persistence-storage/

If you have better suggestions, welcome to follow the discussion, I will release part of the code later

**Reply by alexmathew11** [reply]
I mate I am talking about 1.0 not 2.0 you have given me the links of 2.0 . So give me the links of zepp os 1.0

**Reply by XiaomaiTX** [reply]
The idea is the same, you can refer to the 1.0 file system interface

https://docs.zepp.com/docs/1.0/reference/device-app-api/hmFS/open/

**Reply by alexmathew11** [reply]
Can you write me the code
